server/mscalendar/calendar.go

```diff
@@ -16,6 +19,8 @@ type Calendar interface {
 	FindMeetingTimes(user *User, meetingParams *remote.FindMeetingTimesParameters) (*remote.MeetingTimeSuggestionResults, error)
 	GetCalendars(user *User) ([]*remote.Calendar, error)
 	ViewCalendar(user *User, from, to time.Time) ([]*remote.Event, error)
+	InitPolling(user *User) (events []*remote.Event, deltaURL string, err error)
```

Comment:
Can we remove the named return types from these two new methods?  There are several occurrences in this file.

Reply:
I wanted to be clear that the `deltaURL` is what is being returned for the `string` return value, so a given dev will know what to expect when the method is called, or when reading godocs. I'm 0/5 on keeping the named return values, as there is now some inconsistency here, and the method(s) will likely not be called except in the recurring job we create. What do you think @jfrerich?

Reply:
I'm opting for removing the `events` and `deltaURL` return values, because they are never used anyway. We can add them later if they need to be returned.